documentation/modules/ROOT/pages/connectors/mongodb.adoc

```diff
@@ -259,6 +225,30 @@ The connector uses the logical name in a number of ways: as the prefix for all t
 You should give each MongoDB connector a unique logical name that meaningfully describes the source MongoDB system.
 We recommend logical names begin with an alphabetic or underscore character, and remaining characters that are alphanumeric or underscore.
 
+// Type: concept
+// Title: How {prodname} MongoDB connectors perform offset consolidation
+// ModuleID: how-debezium-mongodb-connectors-perform-offset-consolidation
+[[mongodb-offset-consolidation]]
+=== Offset consolidation
+The {prodname} MongoDB connector no longer supports `replica_set` connections to sharded MongoDB deployments. 
```

Comment:
```suggestion

The {prodname} MongoDB connector no longer supports `replica_set` connections to sharded MongoDB deployments. 
```
Insert a blank line after the heading to prevent intermittent problems in the downstream dos in which paragraphs the immediately follow headings are swallowed up and fail to render.